### ISSUE TITLE
Can't change colorscheme in MacVim

### ISSUE THREAD

**Issue by antaed** [itchyny/lightline.vim#308]
Hi,
I'm a happy lightline user on my Windows PC but for some reason I cannot change the default colorscheme on my new Mac. My macvim config:
- ~/.gvimrc
- all plugins (including lightline) are installed with minpac in ~/.vim/pack/minpac/

In .gvimrc:
- let g:lightline.colorscheme = 'nord' is placed before colorscheme my-theme-name
- I have set laststatus=2

Thanks!

**Comment by itchyny** [comment]
I'm not sure hat you mean by change the default colorscheme. Please upload some screenshots and describe where it does not work well.

**Comment by antaed** [comment]
For example, as you can see below, I have set the 'nord' color scheme with no effect. I tried other themes as well but the colors never change. Of course, I restarted vim after each attempt.

![screen shot 2018-06-19 at 15 53 19](https://user-images.githubusercontent.com/30736986/41598549-28380036-73d9-11e8-9b0f-1ffdd7e65995.png)

**Comment by itchyny** [comment]
Please confirm `:echo g:lightline.colorscheme` prints `nord` and `:message` prints no errors.

**Comment by antaed** [comment]
I confirm :echo (...) printed nord, :message prints as below:
![screen shot 2018-06-19 at 16 17 20](https://user-images.githubusercontent.com/30736986/41599651-82c177be-73dc-11e8-9146-732a679fb66c.png)

**Comment by itchyny** [comment]
Can you paste your vimrc here? Delete unrelated lines and plugins if you can.

**Comment by antaed** [comment]
Not sure what is relevant and what not:
[gvimrc_truncated.txt](https://github.com/itchyny/lightline.vim/files/2115387/gvimrc_truncated.txt)
[gvimrc_full.txt](https://github.com/itchyny/lightline.vim/files/2115389/gvimrc_full.txt)

**Comment by itchyny** [comment]
Can you test without vim-session plugin?

**Comment by itchyny** [comment]
Umm, I think Colorscheme event is emitted before .gvimrc is sourced. So I think you can place the config to .vimrc. A bit wired and looking into the Vim source code now.

**Comment by itchyny** [comment]
Okay, I think the cause is written in `macvim-colorscheme`.  Another workaround is writing `let macvim_skip_colorscheme = 1` in .vimrc.

**Comment by antaed** [comment]
Not at work anymore and I don't have a Mac at home to test. I'll get back tomorrow, thank you very much for your help.

**Comment by itchyny** [comment]
I think I have to take some time to fix in the plugin.

**Comment by antaed** [comment]
Ok, no worries. Thanks again for your help. Have a nice day!

**Comment by itchyny** [comment]
This should be fixed now. You can write the config in gvimrc.

**Comment by antaed** [comment]
It is fixed indeed, thanks again.
I still can't add my own theme. I can only load it if I put the code in one of the existing colorscheme files.

**Comment by itchyny** [comment]
Can you confirm `:echo g:lightline#colorscheme#antaed#palette` prints the colorscheme?

**Comment by antaed** [comment]
I get:
E121: Undefined variable: g:lightline#colorscheme#antaed#palette
E15: Invalid expression: g:lightline#colorscheme#antaed#palette

**Comment by itchyny** [comment]
Where do you define that variable? You can define in `{one of the paths in &rtp}/autoload/lightline/colorscheme/antaed.vim`.

**Comment by antaed** [comment]
The variable is defined in antaed.vim, which is placed in the same folder with the rest of the lightline colorshemes:
~/.vim/pack/minpac/start/lightline.vim/autoload/lightline/colorscheme/

Moreover, I even tried to duplicate one of the existing themes, rename it to antaed.vim and modify the variable (ex. from ...#nord#... to ...#antaed#...) but it still doesn't work. 

To make my theme work I did the opposite: I renamed antaed.vim to nord.vim and modified the variable accordingly. So I load my custom theme with ` let g:lightline.colorscheme = 'nord' ` in .gvimrc.
